mentoring/programs/office-hours.md

```diff
@@ -0,0 +1,35 @@
+title: Office Hours/ Mentoring
+weight: 4
+description: Office hours of different SIGs
+---
+
+* API Machinery - 
+    * [Kubebuilder and Controller Runtime Meeting - Second Wednesday every month at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1Ih-2cgg1bUrLwLVTB9tADlPcVdgnuMNBGbUl4D-0TIk/edit?usp=sharing)
+* Architecture - 
+    * [Enhancements Subproject Meeting - Biweekly Tuesday at 08:30 PT (Pacific Time)](https://bit.ly/k8s-enhancements-agenda)
```

Comment:
```suggestion
```
This one is a focused meeting and not really intended for mentoring / office hours